ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/StructLog.java

```diff
@@ -153,4 +148,39 @@ public int hashCode() {
     result = 31 * result + Arrays.hashCode(stack);
     return result;
   }
+
+  public static String toCompactHex(final Bytes abytes, final boolean prefix) {
+    if (abytes.isEmpty()) {
+      if (prefix) return "0x0";
+      else return "0";
+    }
+    final char[] hexChars = HEX_DIGITS.toCharArray();
```

Comment:
I think this can be a constant char[] rather than allocating a new array on each invocation

Reply:
Done